templates/settings.yaml.erb

```diff
@@ -3,11 +3,7 @@
 ---
 <%= @header %>
 
-#your default puppet server - can be overridden in the host level
```

Comment:
Maybe a nitpick, but maybe this should be a separate commit? I'm in favour of removing it BTW.